scripts/display-travis-status.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const debug = require('debug')('display_travis_status')
+const pollTravis = require('../lib/pollTravis')
+
+module.exports = function (app) {
+  app.on('pull_request.opened', (event) => {
+    const owner = event.repository.owner.login
+    const repo = event.repository.name
+    if (repo !== 'nodejs.org') return
```

Comment:
Just to make one thing clear; I'll bring it up on a meeting and see if there are objections, but since we've been talking about writing software that polls CI and pings back for a year I don't see how it wouldn't be accepted. 

I'll put together my thoughts on how this needs to work which may or may not lead to a PR.